### PR TITLE
Add SQLDumpRestoration.md file

### LINKED ISSUE #993: Add SQLDumpRestoration.md file
Add the documentation about restoring the database from a SQL Dump.

### PR DESCRIPTION
**Pull request by Kirandawadi**
Add the documentation about restoring the database from a SQL Dump.

### CODE REVIEW

File: SQLDumpRestoration.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+5. Transfer the backup SQL dump (`backup.sql`) from your local machine to the PostgreSQL container.
+
+```
+docker cp /local/path/backup.sql 23d33f22cc43:/
+```
+
+6. Import the SQL dump into the PostgreSQL container.
+
+```
+psql -U VnUvMKBSdkoFIETgLongnxYHrYVJKufn -d sde_indexing_helper -f backup.sql
```
Comment (code-geek) [inline SQLDumpRestoration.md]: Can you replace `VnUvMKBSdkoFIETgLongnxYHrYVJKufn` with `<POSTGRES_USER> (.envs/.local/.postgres)`, and `sde_indexing_helper` with `<POSTGRES_DB> (.envs/.local/.postgres)` ?

File: SQLDumpRestoration.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+**Note**: To create a SQL dump of your PostgreSQL database, use the following command:
+
+```
+pg_dump -U VnUvMKBSdkoFIETgLongnxYHrYVJKufn -W -F p -f backup.sql sde_indexing_helper
```
Comment (code-geek) [inline SQLDumpRestoration.md]: Can you replace `VnUvMKBSdkoFIETgLongnxYHrYVJKufn` with `<POSTGRES_USER> (.envs/.production/.postgres)`, and `sde_indexing_helper` with `<POSTGRES_DB> (.envs/.production/.postgres)` ?

File: SQLDumpRestoration.md
Comment (code-geek) [inline SQLDumpRestoration.md:1]: Can you link this file to the main README.md in the appropriate section?

**Review by code-geek — COMMENTED** [review]
Great job putting this together. A few minor comments.

File: SQLDumpRestoration.md (hunk 2 above)
Comment (Kirandawadi) [reply SQLDumpRestoration.md]: Made the mentioned changes.

File: SQLDumpRestoration.md
Comment (Kirandawadi) [reply SQLDumpRestoration.md:1]: Linked the file to the appropriate section in README.md.

**Review by code-geek — APPROVED** [review]
LGTM

**Pull request merged**